content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -9,15 +9,42 @@ menu:
     parent: About the project
 ---
 
-## v1.9.4 []
+## v1.9.4 [TK]
 
 ### Features
-
-- Added `memUsage` metrics to `/debug/vars` endpoint.
-  Measures memory usage in bytes across all subscriptions.
-- Make `compact-full-write-cold-duration` apply to both TSM files and the TSI index.
+- Add more options for [restoring](/enterprise_influxdb/v1.9/tools/influxd-ctl/#restore) Enterprise databases.
+  Users can now:
+  restore data with a new retention policy into an existing database,
+  override the duration of a retention policy while restoring, and
+  specify a destination shard when restoring a specific shard. <!-- https://github.com/influxdata/plutonium/pull/3608 -->
+- Add `memUsage` metrics to [`/debug/vars`](/enterprise_influxdb/v1.9/tools/api/#debugvars-http-endpoint) endpoint
+  to measure memory usage in bytes across all subscriptions.<!-- https://github.com/influxdata/influxdb/pull/22020 -->
+- Allow specification and filtering of `SHOW TAG VALUES` by retention policy. <!-- https://github.com/influxdata/influxdb/pull/22011 -->
+- Improve memory performance by making `compact-full-write-cold-duration` apply to both TSM files and the TSI index.
+- Update Protocol Buffers library versions. <!-- https://github.com/influxdata/influxdb/pull/21890 -->
+- Update to Flux [0.127.3](/influxdb/v2.0/reference/release-notes/flux/). <!-- https://github.com/influxdata/influxdb/pull/22102 -->
```

Comment:
```suggestion
- Update to Flux [0.127.3](/flux/v0.x/release-notes/#v01273-2021-09-01). <!-- https://github.com/influxdata/influxdb/pull/22102 -->
```

Reply:
There are currently no release notes for v0.127.3.